src/Microsoft.AspNetCore.SystemWebAdapters.SessionState/RemoteSession/Framework/GetWriteableSessionHandler.cs

```diff
@@ -51,7 +51,9 @@ public async Task ProcessRequestAsync(HttpContextBase context, CancellationToken
 
         // Delimit the json body with a new line to mark the end of content
         context.Response.OutputStream.WriteByte(EndOfFrame);
-        await context.Response.OutputStream.FlushAsync(token);
+
+        // Ensure to call HttpResponse.FlushAsync to flush the request itself, and not context.Response.OutputStream.FlushAsync()
+        await context.Response.FlushAsync();
```

Comment:
I think this was changed based on an analyzer run previously. Is there a diagnostic we need to suppress here or anything like that?

Reply:
I changed it figuring it was a similar API and thought it would be easier to test operating directly on stream. It was a last minute change that I hadn't run the actual E2E :( No analyzers for this that I'm aware of